doc/api/sqlite.md

```diff
@@ -496,9 +512,38 @@ An object containing commonly used constants for SQLite operations.
 
 The following constants are exported by the `sqlite.constants` object.
 
-#### Conflict-resolution constants
+#### Conflict resolution constants
+
+One of the following constants is available as an argument to the `onConflict` conflict resolution handler passed to [`database.applyChangeset()`](#databaseapplychangesetchangeset-options)). See also [Constants Passed To The Conflict Handler](https://www.sqlite.org/session/c_changeset_conflict.html) in the SQLite documentation.
```

Comment:
Nit: line wrapping here and ideally move the anchor references to the section at the end of the file

Reply:
Done. https://github.com/nodejs/node/pull/56352/commits/c4a03b5e37fa8f419e3dfc0aa103c49084de3416